### PR TITLE
Tips for SDK on Fedora 37

### PR DESCRIPTION
**Pull request by Tomin1**
Add openssl 1.1 requirement to install instructions.

Add workaround for stuck builds to known issues.

### CODE REVIEW

File: Tools/Sailfish_SDK/Known_Issues/README.md [hunk 1]
```diff
@@ -54,6 +54,12 @@ These are the known issues with the Sailfish SDK. If you have any questions, fee
       - Sailfish IDE needs restart after changing SSH port of a Docker based build engine
       - Error starting the build engine on [Linux hosts with cgroups v2](https://forum.sailfishos.org/t/issues-with-installing-sailfish-sdk-docker-with-debian-bullseye-11/4454)
           - It is usually possible to revert back to cgroups v1 by adding `systemd.unified_cgroup_hierarchy=false` to the host kernel command line
+      - On some systems package manager gets stuck or crashes due to unreasonably high file descriptor limit and builds will not get past initialization phase. You may stop build engine with `sfdk engine stop` and then use this workaround:
+        ```
+    sfdk engine exec
```
Comment (Tomin1) [inline Tools/Sailfish_SDK/Known_Issues/README.md:59]: I guess these could be combined to a single line. Improvement suggestions welcome. :slightly_smiling_face:

Comment (martyone) [reply Tools/Sailfish_SDK/Known_Issues/README.md:59]: yup, `echo ... |sfdk engine exec sudo tee ...` should work as well. Or maybe go multiline but still single command with

```
sfdk engine exec sudo tee /etc/security/limits.d/95-nofile-sdk-fix.conf <<END
# This is a workaround for issues with package manager
* hard nofile 4096
root hard nofile 4096
END
```

**Review by martyone — APPROVED** [review]
LGTM, feel free to refactor

File: Tools/Sailfish_SDK/Known_Issues/README.md (hunk 1 above)
Comment (Tomin1) [reply Tools/Sailfish_SDK/Known_Issues/README.md]: Multiline is a very nice idea, thanks! :+1: I changed this to use that and limited the length of the lines above to something a bit more reasonable.

Comment (martyone) [reply Tools/Sailfish_SDK/Known_Issues/README.md]: oops, it's desired *not* to wrap long lines

Comment (Tomin1) [reply Tools/Sailfish_SDK/Known_Issues/README.md]: Ah. I didn't know that. I'll change those back then.

**Review by martyone — APPROVED** [review]

**Pull request merged**